packages/opentelemetry-core/src/common/time.ts

```diff
@@ -163,6 +199,7 @@ export function isTimeInput(
   return (
     isTimeInputHrTime(value) ||
     typeof value === 'number' ||
+    typeof value === 'bigint' ||
```

Comment:
This is related to my question on api/src/common/Time.ts about whether the intend is to allow BigInt (presumably always representing timeUnixNanos) in `TimeInput`.